apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -168,32 +93,10 @@ function AppInitializer(props: { children?: React.ReactNode }) {
     }
   }, [appId])
 
-  /** Call getSession on mount */
-  useEffect(() => {
-    getSession()
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
-
   useAnalytics()
 
-  useDetectSSOCode({
```

Comment:
useDetectSSOCode, which handles receiving the redirect when signing in with Google, is now handled in the `ApplicationSessionManager`